lib/svgo/js2svg.js

```diff
@@ -28,6 +28,8 @@ var defaults = {
   encodeEntity: encodeEntity,
   pretty: false,
   useShortTags: true,
+  eol: undefined,
```

Comment:
Let's just pass `\n` or `\r\n` here

Reply:
In other words, `platformEOL` goes here?

Reply:
yep

Reply:
Addressed in latest push.